### PR TITLE
Release 2022.12.0 Release Notes and Updates

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding monthly release notes and associated doc updates as per December release.
- **Is this PR related to a Monolith release?** 2022.12.0

### Resolves:

* [OKTA-553143](https://oktainc.atlassian.net/browse/OKTA-553143)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 1]
```diff
@@ -423,6 +424,118 @@ Content-Type: application/json;charset=UTF-8
...
+| invalid_scope            | The scopes list contains an invalid or unsupported value.                                                                                                                                                  |
+| unknown_user_id          | The OpenID Provider isn't able to identify which user the client wants authenticated by means of the hint provided in the request.                                                                         |
+| unauthorized_client      | The client isn't authorized to use this authentication flow.                                                                                                                                               |
+| access_denied            | The server denied the request.                                                                                                                                                                             |
+
+#### CIBA request examples
+
+###### Request with `login_hint`
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:488]:
```suggestion
##### Request with `login_hint`
```

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 2]
```diff
@@ -423,6 +424,118 @@ Content-Type: application/json;charset=UTF-8
...
+
+```bash
+curl -v -X POST \
+-H "Content-type:application/x-www-form-urlencoded" \
+-H "User-Agent: Mozilla/5.0 (${systemInformation}) ${platform} (${platformDetails}) ${extensions}" \
+"https://${yourOktaDomain}/oauth2/default/v1/bc/authorize" \
+-d "client_id=${clientId}&client_secret=${clientSecret}&scope=${scope}&binding_message=${binding_message}&login_hint=${login_hint}"
+```
+
+###### Request with `id_token_hint`
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:498]:
```suggestion
##### Request with `id_token_hint`
```

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 3]
```diff
@@ -423,6 +424,118 @@ Content-Type: application/json;charset=UTF-8
...
+
+```bash
+curl -v -X POST \
+-H "Content-type:application/x-www-form-urlencoded" \
+-H "User-Agent: Mozilla/5.0 (${systemInformation}) ${platform} (${platformDetails}) ${extensions}" \
+"https://${yourOktaDomain}/oauth2/default/v1/bc/authorize" \
+-d "client_id=${clientId}&client_secret=${clientSecret}&scope=${scope}&binding_message=${binding_message}&id_token_hint=${id_token_hint}"
+```
+
+###### Request with signed `request`
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:508]:
```suggestion
##### Request with signed `request`
```

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 4]
```diff
@@ -441,7 +554,7 @@ The following parameters can be posted as a part of the URL-encoded form values
...
+| grant_type              | Can be one of the following: `authorization_code`, `password`, `client_credentials`, `refresh_token`, `urn:ietf:params:oauth:grant-type:device_code`<ApiLifecycle access="ea" />, `urn:ietf:params:oauth:grant-type:token-exchange`, `urn:ietf:params:oauth:grant-type:saml2-bearer`<ApiLifecycle access="ea" />, or `urn:openid:params:grant-type:ciba` <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />. Determines the mechanism Okta uses to authorize the creation of the tokens.                                                                                                                                               | String |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:557]:
```suggestion
| grant_type              | Can be one of the following: `authorization_code`, `password`, `client_credentials`, `refresh_token`, `urn:ietf:params:oauth:grant-type:device_code`, `urn:ietf:params:oauth:grant-type:token-exchange`, `urn:ietf:params:oauth:grant-type:saml2-bearer`, or `urn:openid:params:grant-type:ciba` <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />. Determines the mechanism Okta uses to authorize the creation of the tokens.                                                                                                                                               | String |
```

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 5]
```diff
@@ -918,29 +1032,31 @@ curl -X GET \
...
+| ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| issuer                                                            | The complete URL for a custom authorization server. This becomes the `iss` claim in an access token. In the context of this document, this is your authorization server's [base URL](#composing-your-base-url).                                                                                                                                              | String  |
+| authorization endpoint                                            | URL of the authorization server's [authorization endpoint](#authorize).                                                                   | String  |
+| device_authorization_endpoint                                     | URL of the authorization server's [device authorize endpoint](#device-authorize)<ApiLifecycle access="ea" />.                                                                                                                      | String   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1039]:
```suggestion
| device_authorization_endpoint                                     | URL of the authorization server's [device authorize endpoint](#device-authorize).                                                                                                                      | String   |
```

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 6]
```diff
@@ -1074,29 +1205,31 @@ curl -X GET \
...
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- |
+| authorization_endpoint                                            | URL of the authorization server's [authorization endpoint](#authorize).                                                                                                                                                                    | String  |
+| device_authorization_endpoint                                     | URL of the authorization server's [device authorize endpoint](#device-authorize)<ApiLifecycle access="ea" />.                                                                                                                      | String   |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:1211]:
```suggestion
| device_authorization_endpoint                                     | URL of the authorization server's [device authorize endpoint](#device-authorize).                                                                                                                      | String   |
```

**Review by susanharper-okta — APPROVED** [review]
couple things - primarily removing more EA flags. Approving to not hold you up.

**Comment by brianduffield-okta** [comment]
Release to Okta Preview 1 at 5:08 AM

**Pull request merged**